src/helpers/figures/charts/pyramid_chart.ts

```diff
@@ -179,8 +187,36 @@ export class PyramidChart extends AbstractChart {
     };
   }
 
-  getDefinitionForExcel(): ExcelChartDefinition | undefined {
-    return undefined;
+  getDefinitionForExcel(getters: Getters): ExcelChartDefinition | undefined {
+    if (this.aggregated) {
+      // Excel does not support aggregating labels
+      return undefined;
+    }
+    const dataSets: ExcelChartDataset[] = this.dataSets
+      .map((ds: DataSet) => toExcelDataset(this.getters, ds))
+      .filter((ds) => ds.range !== "" && ds.range !== CellErrorType.InvalidReference);
+    const labelRange = toExcelLabelRange(
+      this.getters,
+      this.labelRange,
+      shouldRemoveFirstLabel(this.labelRange, this.dataSets[0], this.dataSetsHaveTitle)
+    );
+    const definition = this.getDefinition();
+    const chartData = getPyramidChartData(definition, this.dataSets, this.labelRange, getters);
+    const { dataSetsValues } = chartData;
+    const maxValue = Math.max(
+      ...dataSetsValues.map((dataSet) => Math.max(...dataSet.data.map(Math.abs)))
+    );
+    return {
+      ...definition,
+      horizontal: true,
+      backgroundColor: toXlsxHexColor(this.background || BACKGROUND_CHART_COLOR),
+      fontColor: toXlsxHexColor(chartFontColor(this.background)),
+      dataSets,
+      labelRange,
+      verticalAxis: getDefinedAxis(definition),
+      maxValue,
+      minValue: -maxValue,
```

Comment:
nitpick: we'd expect `minValue` to be the smallest value present in the chart data, not `-maxValue`. I'd just remove `minValue` from the excel chart definition, you can just compute `-maxValue` when creating the xlsx 🙂